tests/test_trusted-execution.yml

```diff
@@ -0,0 +1,132 @@
+---
+- name: Basic sanity test for fapolicyd
+  hosts: localhost
+  vars:
+    binaries_directory: "/var/tmp/executable_binaries"
+    source_directory: "{{ binaries_directory }}/source"
+    source_file: "{{ source_directory }}/main.c"
+    exe1: "{{ binaries_directory }}/exe1"
+    exe2: "{{ binaries_directory }}/exe2"
+    user: "testuser"
+
+  tasks:
+    - name: Create {{ binaries_directory }} directory
```

Comment:
Ansible style is to not have variables in the name - if you really want to use them, the variable must be the last thing in the name - so either
```
    - name: Create binaries_directory directory
```
or
```
    - name: Create directory {{ binaries_directory }}
```